src/Stratis.SmartContracts.Executor.Reflection/ExternalExecutor.cs

```diff
@@ -12,23 +12,23 @@ namespace Stratis.SmartContracts.Executor.Reflection
     /// <summary>
     /// Deserializes raw contract transaction data, creates an external create/call message, and applies the message to the state.
     /// </summary>
-    public class Executor : ISmartContractExecutor
+    public class ExternalExecutor : IContractExecutor
```

Comment:
I'm wondering whether we shouldn't call this `ContractExecutor` perhaps?

Reply:
I like this suggestion, will update